Aspose.Slides.WebExtensions/Helpers/CssNumbersExtension.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Aspose.Slides.WebExtensions.Helpers
+{
+    public static class CssNumbersExtension
+    {
+        public const int DEFAULT_PRECISION = 3;
+        public static string ToCssNumber(this float number, int precision = DEFAULT_PRECISION, bool trimEndZeros = true)

```

Comment:
@polonair ,

Please do not use extension methods in Util classes (we finally should support this in the main Slides assembly). Same about default parameter values.